scripts/README.md

```diff
@@ -0,0 +1,11 @@
+# Scripts
+
+Here a list of scripts we are using to make our life easier.
+
+*Please note that those scripts aren't bulletproof nor all have the best code we could have done: they were created quickly (we plan to make them better at some point—meaning probably never), so you are using them at your own risk*
```

Comment:
I just tweaked the sentence a bit: Please note that those scripts aren't bulletproof _nor do they_ have the best code we could have done: